File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/commands/UpdateSeatType.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class UpdateSeatType : Command<Guid>
    {
        public Guid SeatTypeId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }

        public UpdateSeatType() { }
        public UpdateSeatType(Guid conferenceId) : base(conferenceId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/UpdateSeatType.java
package com.microsoft.conference.common.management.commands;

import org.enodeframework.commanding.Command;

import java.math.BigDecimal;

public class UpdateSeatType extends Command<String> {
    public String seatTypeId;
    public String name;
    public String description;
    public BigDecimal price;
    public int quantity;

    public UpdateSeatType() {
    }

    public UpdateSeatType(String conferenceId) {
        super(conferenceId);
    }
}
